File: src/main/java/de/saschahlusiak/freebloks/view/model/ViewElement.java
package de.saschahlusiak.freebloks.view.model;

import android.graphics.PointF;

public interface ViewElement {
	boolean handlePointerDown(PointF m);
	boolean handlePointerMove(PointF m);
	boolean handlePointerUp(PointF m);
	boolean execute(float elapsed);
}

File: src/main/java/de/saschahlusiak/freebloks/view/model/ViewElement.kt
package de.saschahlusiak.freebloks.view.model

import android.graphics.PointF

interface ViewElement {
    fun handlePointerDown(m: PointF): Boolean
    fun handlePointerMove(m: PointF): Boolean
    fun handlePointerUp(m: PointF): Boolean

    fun execute(elapsed: Float): Boolean
}